syntax = "proto3";

package zrpc;

message RpcPackage {
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 arg_size = 3;
}